app/src/main/java/com/example/she/ViewContacts.java

```diff
@@ -20,37 +23,65 @@
 
     ListView listView;
     DatabaseHandler db;
-    ArrayList<Pair<String,String>> contactList;
+    ArrayList<Pair<String, String>> contactList;
+    ContactListAdaptor contactListAdaptor;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_view_contacts);
 
+        updateListView();
+    }
+
+    @Override
+    public void editOptionSelected(String phoneNumber) {
+        Toast.makeText(this, "Edit coming soon", Toast.LENGTH_SHORT).show();
+    }
+
+    @Override
+    public void deleteOptionSelected(final String phoneNumber) {
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+         builder.setTitle("Delete")
+                .setMessage("Are you sure to delete?")
```

Comment:
setMessage("Are you sure you want to delete?")
Change the message to this.

Reply:
Ok

Reply:
Done